taco/internal/api/routes.go

```diff
@@ -274,4 +274,7 @@ func RegisterRoutes(e *echo.Echo, deps Dependencies) {
 			}},
 		})
 	})
+
+	// Register webhook-authenticated internal routes (if OPENTACO_WEBHOOK_SECRET is set)
```

Comment:
if you end up changing the flag to OPENTACO_ENABLE_INTERAL_ENDPOINTS then this comment needs updating